pyproject.toml

```diff
@@ -24,6 +24,8 @@ ordered-set = ">=4.1.0"
 typer = "^0.7.0"
 typer-cli = "^0.0.13"
 loguru = "*"
+#sssom = "*"
+sssom = { git = "https://github.com/glass-ships/sssom-py.git", branch = "develop" } 
```

Comment:
Why are you pointing to a fork? If you want to work on a branch, we can just give you write access to the main project!

Reply:
I gave you write access.